config/producer.properties

```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# see kafka.producer.ProducerConfig for more details
+# see org.apache.kafka.clients.producer for more details
```

Comment:
I missed the first time around, but `ProducerConfig` is missing.

Reply:
Ahh.sorry for this..raised another PR #4070